static/src/javascripts/projects/common/modules/identity/cookierefresh.js

```diff
@@ -0,0 +1,30 @@
+// @flow
+
+import {
+    isUserLoggedIn,
+    getUserFromApiWithRefreshedCookie,
+} from 'common/modules/identity/api';
+import { local as localStorage } from 'lib/storage';
+
+const shouldRefreshCookie = (
+    lastRefresh: ?number,
+    currentTime: number
+): boolean => {
+    const days30k = 1000 * 86400 * 30; // (as seconds)
```

Comment:
Not sure what the significance of this is. I'd prefer to go with, e.g. `86400 * 365 * 5` (5 years approx) but it doesn't really matter.

Reply:
Probably no special significance. It was implemented 2.5 years ago without explanation (#9191) 😄